components/HomeHero/HomeHero.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+
+import logo from '../../public/images/headless-logo.svg';
+
+import styles from './HomeHero.module.scss';
+
+import Strapline from './Strapline';
+import type { StraplineProps } from './Strapline/types';
+
```

Comment:
I think we're just using the strapline in the homepage, so you could keep both components in the same file (instead of splitting them into separate files within the same directory).
```
const Strapline = () => { ... }

const HomeHere = () => { ... }
```

The disadvantage to this is you have to be more careful with styling names

Reply:
The disadvantage of splitting the components is that we don't reuse one of them. However, putting them in the same directory has the disadvantage of having longer code files and less structure in my opinion, in addition of the one you mentioned about styling names.

Can't we keep it as is knowing that each file structure has its advantages and disadvantages?

Reply:
Yes - I think in this project, as we're using SCSS modules, keeping the class names simple is slightly more important than the overhead of different file setups. I think I prefer having child components in the same file that aren't reused as I've been working on a project using Tailwind, where there is less confusion with class names.